packages/react-ui-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rentpath/react-ui-core",
-  "version": "19.4.3",
+  "version": "19.5.0",
```

Comment:
Does this repo not auto-bump versions on publish?

Reply:
🤔 

I need to look further into this.